packages/orchestration/src/examples/staking-combinations.flows.js

```diff
@@ -41,43 +45,65 @@ harden(makeAccount);
  * @satisfies {OrchestrationFlow}
  * @param {Orchestrator} orch
  * @param {object} ctx
+ * @param {{ localAccount?: OrchestrationAccountI & LocalAccountMethods }} ctx.contractState
+ * @param {GuestInterface<ZoeTools>} ctx.zoeTools
  * @param {GuestInterface<CosmosOrchestrationAccount>} account
  * @param {ZCFSeat} seat
  * @param {CosmosValidatorAddress} validator
- * @param {AmountArg} amount
  * @returns {Promise<string>}
  */
 export const depositAndDelegate = async (
   orch,
-  ctx,
+  { contractState, zoeTools },
   account,
   seat,
   validator,
-  amount,
 ) => {
-  console.log('depositAndDelegate', account, seat, validator, amount);
-  // TODO deposit the amount
-  await account.delegate(validator, amount);
+  await null;
+  console.log('depositAndDelegate', account, seat, validator);
+  mustMatch(validator, ChainAddressShape);
+  if (!contractState.localAccount) {
+    const agoricChain = await orch.getChain('agoric');
+    contractState.localAccount = await agoricChain.makeAccount();
+  }
+  const { give } = seat.getProposal();
+  await zoeTools.localTransfer(seat, contractState.localAccount, give);
+
+  const address = account.getAddress();
+  try {
+    await contractState.localAccount.transfer(give.Stake, address);
+  } catch (cause) {
+    // TODO, put funds back on user seat and exit
+    // https://github.com/Agoric/agoric-sdk/issues/9925
+    throw makeError('ibc transfer failed', undefined, { cause });
+  }
+  seat.exit();
+  await account.delegate(validator, give.Stake);
   return 'guest depositAndDelegate complete';
 };
 harden(depositAndDelegate);
 
 /**
  * @satisfies {OrchestrationFlow}
- * @param {Orchestrator} orch
- * @param {object} ctx
+ * @param {Orchestrator} _orch
+ * @param {object} _ctx
  * @param {GuestInterface<CosmosOrchestrationAccount>} account
- * @param {Omit<Delegation, 'delegatorAddress'>[]} delegations
+ * @param {{
+ *   delegations: { amount: AmountArg; validator: CosmosValidatorAddress }[];
+ *   destination: ChainAddress;
+ * }} offerArgs
  * @returns {Promise<string>}
  */
 export const undelegateAndTransfer = async (
-  orch,
-  ctx,
+  _orch,
+  _ctx,
```

Comment:
what prompted this renaming? let's only use `_` when linting requires it. It's obvious enough in the editor that it's never used. (same applies to `_privateArgs` change in the contract)

Reply:
we do, to not have extra bindings hanging around. It's usually indicative of a coding error.

but when it's in a list of params and you do use the latter params, binding the earlier ones is necessary. so there's no point in warning someone that they didn't use them

Reply:
Ah I see now - didn't realize this was the case but makes sense. Thanks for explaining!